lib/bug_report_templates/active_admin_master.rb

```diff
@@ -8,22 +8,23 @@
 gemfile(true) do
   source 'https://rubygems.org'
 
-  gem 'rails', require: false
-  gem 'sqlite3', platform: :mri
+  # Use local changes or ActiveAdmin master.
+  if ENV['ACTIVE_ADMIN_PATH']
+    gem 'activeadmin', path: ENV['ACTIVE_ADMIN_PATH'], require: false
```

Comment:
No I didn't know about, neat! I didn't want to change this because its how it works with the spec and also because of time haha. Is that ok? Or you prefer we use that? We can always address separately too.